src/Microsoft.DotNet.CodeFormatting.Tests/CodeFormattingTestBase.cs

```diff
@@ -97,73 +93,4 @@ protected void Verify(string source, string expected, bool runFormatter = true,
         }
 
     }
-
-    // TODO: Pull RuleTestBase and its three derived classes into separate file RuleTestBase.cs.
```

Comment:
The rest of this file is pulled into a separate file RuleTestBase.cs. No other code change (except to remove now-unused usings).